src/partials/contacts.html

```diff
@@ -0,0 +1,66 @@
+<section class="contacts">
+    <div class="contacts__container">
+    <h2 class="contacts__title">Contacts</h2>
+    <ul class="contacts__box">
+        <li class="contacts__card">
+            <p class="contacts__category contacts__category--cafe">CAFE</p>
+            <article class="contacts__place">
+                <h3 class="contacts__city">Chicago</h3>
+                <p class="contacts__description">Fusce ut velit laoreet, tempus arcu molestie vulputate</p>
+            </article>
+            <div class="contacts__opened">
+                <h3 class="contacts__day">Monday - Friday</h3>
+                <p class="contacts__time"><time datetime="06:00">06:00 AM</time> - <time datetime="22:00">10:00 PM</time></p>
+                <h3 class="contacts__day">Saturday - Sunday</h3>
+                <p class="contacts__time"><time datetime="08:00">08:00 AM</time> - <time datetime="16:00">04:00 PM</time></p>
+            </div>
+            <address class="contacts__contact">
+                <a class="contacts__tel" href="tel:+610383766284">+61(0) 383 766 284</a>
+                <a class="contacts__mail" href="mailto:noreply@envato.com">noreply@envato.com</a>
+            </address>
+        </li>
+        <li class="contacts__card">
+            <p class="contacts__category contacts__category--foodtruck">FOOD TRUCK</p>
```

Comment:
zamiast contacts__category--foodtruck wstawił bym contacts__category--larger i w scss dodał bym do niego wielkość można też dodać wtedy do niego kolor ale też możesz stowrzyć odzielną klasę np: contacts__category--accent i do niego wstawić ten jasny zielony kolor 

(chodzi o to aby modifier mówił co się dzieje z tym elementem)

Reply:
zrobione